src/applications/personalization/account/components/AccountMain.jsx

```diff
@@ -45,21 +52,24 @@ class AccountMain extends React.Component {
 
   render() {
     const {
-      profile: {
-        loa,
-        multifactor,
-        verified
-      },
-      terms
-    } = this.props;
+      loa,
+      loading,
+      mhv,
+      multifactor,
+      verified
+    } = this.props.profile;
+
+    if (loading || mhv.account.loading) {
+      return <LoadingIndicator message="Loading your account information..."/>;
+    }
 
     return (
       <div>
         <AccountVerification loa={loa}/>
         {this.renderMVIError()}
         <MultifactorMessage multifactor={multifactor}/>
         <LoginSettings/>
-        <TermsAndConditions terms={terms} verified={verified}/>
+        {verified && <TermsAndConditions mhv={mhv}/>}
```

Comment:
@bshyong @ncksllvn: Tagged for awareness about changes to T&C logic in account page.